http-clients/aws-crt-client/src/test/java/software/amazon/awssdk/http/crt/ProxyConfigurationTest.java

```diff
@@ -76,6 +109,8 @@ private void setRandomValue(Object o, Method setter) throws InvocationTargetExce
             setter.invoke(o, randomString());
         } else if (int.class.equals(paramClass)) {
             setter.invoke(o, RNG.nextInt());
+        } else if (Boolean.class.equals(paramClass)) {
+            setter.invoke(o, Boolean.FALSE);
```

Comment:
I suppose this should be `RNG.nextBoolean()` to reflect the "random" method name.